{{cookiecutter.import_name}}/{{cookiecutter.import_name}}/app/ui.py

```diff
@@ -1,16 +1,19 @@
 from trame import controller as ctrl
 from trame.layouts import SinglePage
 from trame.html import vuetify
+{%- if cookiecutter.include_components %}
 from .. import html as my_widgets
+{%- endif %}
```

Comment:
Should I change this import to `from trame_app import html as my_widgets`?

Reply:
yes but I can do it once I update the trame code with more content